pom.xml

```diff
@@ -62,6 +62,11 @@
     </licenses>
     
     <dependencies>
+        <dependency>
```

Comment:
It pulls in the dependency on some Pipeline API plugins. It does not seem to be a problem since they do not contribute to UI